apps/mobile-wallet/__tests__/test.test.ts

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+it('works', () => {
```

Comment:
Out of curiosity: why is this needed? If tests don't work we'll see them failing somehow, no?

Reply:
haha, yes, of course! But look at the test, do you want this to fail? :P 

```
expect(1).toBe(1)
```

I am simply setting up the test suite here, as I described in the PR I could not find a way to test expo secure store